pallets/operation/src/lib.rs

```diff
@@ -428,6 +430,32 @@ pub mod pallet {
             Self::deposit_event(Event::<T>::BurnForEZC(sender, balance, benifity));
             Ok(().into())
         }
+
+        // TODO: use real weight
+        #[pallet::weight(T::OPWeightInfo::start_withdraw_ezc())]
+        pub fn start_withdraw_ezc(
+            origin: OriginFor<T>,
+            eth_address: H160,
+        ) -> DispatchResultWithPostInfo {
+            let sender = ensure_signed(origin)?;
+            Self::deposit_event(Event::<T>::StartWithdrawEZC(sender, eth_address));
```

Comment:
only send event ?

Reply:
Yes, `get_npow_reward` send a event, bridge listen this event, invoke the contract to compute its npow reward, then call `npow_mint` to give some tokens to the account.